src/modules/party/party.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+
+import { AcbsAuthenticationService } from '../acbs/acbs-authentication.service';
+import { AcbsPartyService } from '../acbs/acbs-party.service';
+import { Party } from './party.interface';
+
+@Injectable()
+export class PartyService {
+  constructor(private readonly acbsAuthenticationService: AcbsAuthenticationService, private readonly acbsPartyService: AcbsPartyService) {}
+
+  async getPartyByIdentifier(partyIdentifier: string): Promise<Party> {
```

Comment:
Should we add `partyIdentifier` argument validation:

* Not empty
* Should be min x length please?

Reply:
@IgnatG was worried in APIM-76 that - because MuleSoft and ACBS don't validate it at the moment - if we add validation then it might break other systems. We discussed it [here](https://github.com/UK-Export-Finance/tfs-api/pull/11#discussion_r1127402383)